core-application/src/main/java/gohigher/application/service/ApplicationQueryService.java

```diff
@@ -21,18 +24,34 @@ public class ApplicationQueryService implements ApplicationQueryPort {
 	private final ApplicationPersistenceQueryPort applicationPersistenceQueryPort;
 
 	@Override
-	public List<CalenderApplicationResponse> findByMonth(CalenderApplicationRequest request) {
-		List<Application> applications = applicationPersistenceQueryPort.findByIdAndMonth(request.getUserId(),
+	public List<CalendarApplicationResponse> findByMonth(CalendarApplicationRequest request) {
+		List<Application> applications = applicationPersistenceQueryPort.findByUserIdAndMonth(request.getUserId(),
 			request.getYear(), request.getMonth());
```

Comment:
```suggestion
		List<Application> applications = applicationPersistenceQueryPort.findByUserIdAndYearAndMonth(request.getUserId(),
			request.getYear(), request.getMonth());
```

매개변수에 Year과 Month가 있어서 위처럼 변경하는건 어떨까요? 혹은 findByIdAndDate? 정도요!